demo/agw/ThumbnailCtrl.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, parent, log):
         self.statusbar = self.CreateStatusBar(2)
         self.statusbar.SetStatusWidths([-2, -1])
         # statusbar fields
-        statusbar_fields = [("ThumbnailCtrl Demo, Andrea Gavana @ 10 Dec 2005"),
+        statusbar_fields = [("ThumbnailCtrl Demo, Michael Eager @ 15 Oct 2020"),
```

Comment:
Dont remove someones copywrite. Just append your own to it.

Reply:
> Dont remove someones copywrite. Just append your own to it.

The original copyright is retained.  This is the status line for the demo.

Reply:
@infinity77 Is this Ok with you? ... or how would you deal with this type of change...? It is status text.

Reply:
I don’t have a problem, as long as my name is mentioned somewhere in the library 😀. I think @eagerm has done a very good job in refactoring ThumbnailCtrl into a more usable component